app/(white_navbar)/explore/RetroPGF4/RetroType4.d.ts

```diff
@@ -27,25 +27,25 @@ export interface Github {
   openSourceRejectComment: string
 }
 export interface ImpactMetrics {
-  project_name: string
-  application_id: string
-  is_oss: boolean
```

Comment:
This is because this type was used in `ImpactMetricSection.tsx` (Impact metric in Retro Funding 4 project details) which requires those keys but for this task, I only need all metric keys without these keys (others remain the same keys)

In this task, I have implemented `Partial<ImpactMetrics[]>`, if you think this is not proper way, please let me know :)

Reply:
I believe `Partial<ImpactMetrics[]>`